mmdet/apis/ote/apis/detection/task.py

```diff
@@ -454,6 +456,13 @@ def export(self,
             except Exception as ex:
                 raise RuntimeError("Optimization was unsuccessful.") from ex
 
+    def optimize(self,
+        optimization_type: OptimizationType,
+        dataset: Optional[Dataset],
+        output_model: OptimizedModel,
+        optimization_parameters: Optional[OptimizationParameters],
+    ):
+        pass
```

Comment:
What is it added for?

Reply:
It was done in "example". I also don't sure it's necessary

Reply:
Please remove.

Reply:
Done